GLAA.Web/Controllers/LicenceController.cs

```diff
@@ -461,6 +461,15 @@ public IActionResult SubmitApplication()
             var licenceId = Session.GetCurrentLicenceId();
             var model = LicenceApplicationViewModelBuilder.Build(licenceId);
 
+            LicenceApplicationViewModelBuilder.BuildCountriesFor(model.PrincipalAuthority);
+
+            model.DirectorOrPartner.DirectorsOrPartners =
```

Comment:
This could do with being more efficient, we may need to rethink the approach for this if it causes any performance problems.  The list of countries is however very static and quite small, so it may be OK.  Depends how many DOPs and ABRs we are likely to expect.

Reply:
Yeah this concerned me when I built a list of licences in the admin section and ended up getting the countries once per record.